assets/css/main.css

```diff
@@ -72,21 +72,26 @@ footer {
   padding: 20px 0 20px 0;
 }
 
-.SocialIcons{
+.SocialIcons {
   display: inline-flex;
   justify-content: center;
 }
 
-.SocialIcons a{
+.SocialIcons a {
   width:30px;
 }
 
-.glyphicon, .fa{
+.glyphicon, .fa {
   font-size: 25px!important;
   margin: 5px;
   color:#F5C8D5;
 }
 
-.glyphicon:hover, .fa:hover{
+.glyphicon:hover, .fa:hover {
   color: #fff;
```

Comment:
Every time you mention CSS property, please leave a space before mentioning the value.